prow/plugins/assign/assign.go

```diff
@@ -232,14 +226,13 @@ func newAssignHandler(e *event, gc githubClient, log *logrus.Entry) *handler {
 
 func newReviewHandler(e *event, gc githubClient, log *logrus.Entry) *handler {
 	addFailureResponse := func(mu github.MissingUsers) string {
-		return fmt.Sprintf("GitHub didn't allow me to request PR reviews from the following users: %s.\n\nNote that only people with write access to %s/%s can review this PR.\n", strings.Join(mu.Users, ", "), e.org, e.repo)
+		return fmt.Sprintf("GitHub didn't allow me to request PR reviews from the following users: %s.\n\nNote that only people with write access to %s/%s can review this PR and the PR author cannot review their own PR.\n", strings.Join(mu.Users, ", "), e.org, e.repo)
```

Comment:
nit: s/only people/only other people/ might be a less verbose way to say the same thing